internal/exec/stages/disks/luks.go

```diff
@@ -531,3 +559,87 @@ func (d LuksDump) hasFlag(flag string) bool {
 	}
 	return false
 }
+
+// collect the cex card domains xx.xxxx,
+func cexCardDomain() (string, error) {
+	var ret []string
+	domain, err := exec.Command(distro.LszcryptCmd(), "-V", "--ccaonly", "--queueonly").CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("lszcrypt: No crypto card devices found: %w", err)
```

Comment:
`"lszcrypt: No crypto card devices found: "` I used  the `stderr` when there is no cex card configured in the vm. 
Is it something like `"The system  cannot not  find the cryptographic adapters for secure key generation."`

Reply:
The only thing we know for sure is that the call to `lszcrypt` failed (e.g. it could be that `lszcrypt` isn't in the initrd, or that it couldn't access what it needs in sysfs, or that indeed there are no cards available, etc...). So a more accurate error would be something like:

> failed to list crypto cards using lszscript: %w